scripts/prepdocs.py

```diff
@@ -40,6 +41,15 @@
 CACHE_KEY_CREATED_TIME = 'created_time'
 CACHE_KEY_TOKEN_TYPE = 'token_type'
 
+#Embedding batch section
+AOAI_EMBEDDING_MODEL = 'text-embedding-ada-002'
+MAX_EMB_TOKEN_LIMIT = 8100
```

Comment:
Should this change depending on the embeddings model used?

Reply:
This was marked as resolved but I didn't see a resolution? Is this model-dependent? Do we need a dict for other ada model?